### ISSUE TITLE
content: seed the layer beneath the transformer

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#290]
Following the same lens as #288 — what does a reader arrive already half-knowing — down one level, into what they were taught before any of this. Eleven terms checked, **eleven absent**:

```
activation-function        recurrent-neural-network
long-short-term-memory     vanishing-gradient
batch-normalization        weight-initialization
data-augmentation          early-stopping
transfer-learning          sequence-to-sequence
```

Two of those are pointed at by nodes that already exist and cannot land. **`layer-normalization` is in the corpus and `batch-normalization` is not** — the thing it is named against, and the thing every reader met first. `recurrent` appears in eight bodies and is never defined, so the reader who wants to know what transformers replaced has eight mentions and nowhere to go.

`vanishing gradient` appears in exactly one file, in passing. It is the reason gating exists, the reason residual connections exist, and the reason depth was hard for a decade — and the corpus has `residual-connection` and `gradient-clipping` without the problem they solve.

## Scope — eight nodes

`activation-function` · `recurrent-neural-network` · `long-short-term-memory` · `vanishing-gradient` · `batch-normalization` · `weight-initialization` · `data-augmentation` · `early-stopping`

Scoping constraints against nodes that exist:

- `batch-normalization` must be scoped against `layer-normalization`, which is in the corpus: the axis normalised over is the whole difference, and it is why one works with a batch of one and the other does not.
- `activation-function` must say why a nonlinearity is required at all — a stack of linear layers is one linear layer — before it says anything about which one.
- `early-stopping` must not restate `overfitting`. That node is the phenomenon; this is the intervention, and its subject is that the stopping point is chosen on data you then cannot use to estimate performance.

Several of these predate arXiv and should carry `origin` per ADR-0013 — Hochreiter & Schmidhuber 1997 for LSTM, Bengio, Simard & Frasconi 1994 for the vanishing gradient, Elman 1990 for recurrence.

## Acceptance criteria

- [ ] Eight nodes, schema-valid, connected — no restatement of `layer-normalization`, `overfitting` or `gradient-clipping`
- [ ] Every arXiv id verified by title before being written; pre-arXiv primary sources in `origin`
- [ ] Figures assigned from arithmetic; captions name their control (`content/schema/README.md`)
- [ ] `make golden` regenerated; all gates green
- [ ] Verified in a second pass that re-derives each math body
- [ ] Teal lobe moved when the verification empties it

Refs #61, #288
